spec/features/admin/view_organisation_list_spec.rb

```diff
@@ -2,26 +2,66 @@
 require 'features/support/sign_up_helpers'
 
 describe 'view list of signed up organisations' do
-  context 'when logged out' do
-    before { visit organisations_path }
+  before do
+    login_as user
+    visit organisations_path
+  end
 
+  context 'when not logged in' do
+    let(:user) { nil }
     it_behaves_like 'not signed in'
   end
 
-  context "when logged in" do
-    before do
-      3.times { create(:organisation) }
+  context 'when logged in as a normal user' do
+    let(:user) { create(:user, :confirmed) }
+
+    it 'redirects me to the landing guidance' do
+      expect(page).to have_content 'Get GovWifi'
+      expect(page).to have_content 'Getting help'
     end
+  end
+
+  context 'when logged in as an admin' do
+    let(:user) { create(:user, :confirmed, admin: true) }
+
+    context 'and one organisation exists' do
+      before do
+        org = create(:organisation, name: "Fake Org", created_at: '1 Feb 2014')
+        10.times { create(:location, organisation: org ) }
+        11.times { create(:ip, location: Location.first) }
+        visit organisations_path
+      end
+
+      it 'shows their name' do
+        expect(page).to have_content 'Fake Org'
+      end
 
-    let!(:user) { create(:user, :confirmed, admin: true) }
+      it 'shows when they signed up' do
+        expect(page).to have_content('1 Feb 2014')
+      end
+
+      it 'shows they have 10 locations' do
+        expect(page).to have_content('10')
```

Comment:
Testing this on lower numbers gives false positives, as the page has numbers all over it in dates etc.

I'd switch this to checking more specific parts of the page (e.g. a cell with some class), even if it's a bit brittle, just didn't get to it.